src/intro/setup.md

```diff
@@ -64,9 +108,10 @@ clang -v
 
 
 [compatibility]: ../advanced/compatibility.md
-[godot-download]: https://godotengine.org/download
+[godot-download]: https://godotengine.org/download/archive/
```

Comment:
Maybe the archive should be listed separately from the download page, in parentheses.